src/Screens/SignupScreen.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useState } from 'react';
+import { View, Text, TextInput, ScrollView, StyleSheet, TouchableOpacity, KeyboardAvoidingView, Platform, Image, SafeAreaView } from "react-native";
+import auth from '@react-native-firebase/auth';
+import Colors from '../Contants/Colors';
+
+
+const SignupScreen = (props) => {
+  const [name, setName]=useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [nameError, setNameError] = useState('');
+  const [emailError, setEmailError] = useState('');
+  const [passwordError, setPasswordError] = useState('');
+  const [credientalError, setCredientalError] = useState('');
+
+  const validateName =(name:string)=>{
+    if(!name){
+      setNameError("Enter Your Name");
+      return false;
+    }
+    if(name.length<5){
+      setNameError("Name should be at least 5 characters");
+      return false;
+    }
+    else {
+      setNameError('');
+      return true;
+    }
+  }
+
+  const validateEmail = (email: string) => {

```

Comment:
Please relocate the validateEmail function to the utils module to make it reusable.